### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by adam-soltech**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,13 @@
...
-To run the web server (for development) run: `yarn start`
+*note*: Node 15+ is required and 18+ is recommended. If you run in to an compilation with the node server error where the Buffer object does not have a 'blob' property this is likely the issue. You can check your node version by running 'node -v'
+
+First run the graphql server 'yarn gql-gen'
+
+Next to run the web server (for development) run: `yarn start`
+
+We are patching an issue with openssl. In the interim should you get an openssl error you can run 'export NODE_OPTIONS=--openssl-legacy-provider' from the terminal. To avoid having to do this each time you have a new terminal window you can add that same line to your .zshrc or .bashrc located in your device's user directory ('cd ~')
 
 To view storybook run: `yarn storybook`
```
Comment (CarsonF) [inline README.md:15]: You can delete this, we don't use it anymore.

File: README.md [hunk 2]
```diff
@@ -4,7 +4,13 @@
...
 - `yarn`
 
-To run the web server (for development) run: `yarn start`
+*note*: Node 15+ is required and 18+ is recommended. If you run in to an compilation with the node server error where the Buffer object does not have a 'blob' property this is likely the issue. You can check your node version by running 'node -v'
+
+First run the graphql server 'yarn gql-gen'
+
+Next to run the web server (for development) run: `yarn start`
+
+We are patching an issue with openssl. In the interim should you get an openssl error you can run 'export NODE_OPTIONS=--openssl-legacy-provider' from the terminal. To avoid having to do this each time you have a new terminal window you can add that same line to your .zshrc or .bashrc located in your device's user directory ('cd ~')
```
Comment (CarsonF) [inline README.md:13]: Fixed with #1447 
```suggestion
```

**Review by CarsonF — CHANGES_REQUESTED** [review]
First PR! 🎉

**Comment by adam-soltech** [comment]
Updated

File: README.md [hunk 3]
```diff
@@ -4,7 +4,13 @@
 
 - `yarn`
 
-To run the web server (for development) run: `yarn start`
+*note*: Node 15+ is required and 18+ is recommended. If you run in to an compilation with the node server error where the Buffer object does not have a 'blob' property this is likely the issue. You can check your node version by running 'node -v'
```
[the remark this answers is not in the document]
Comment (adam-soltech) [reply README.md:7]: Ah OK, I thought we would update the readme as we update versions but I guess that's probably wishful thinking

**Review by CarsonF — APPROVED** [review]
LGTM! Thanks :)

**Pull request merged**